tokenizer/lib/spacy/processor.py

```diff
@@ -179,8 +189,7 @@ def _segmentize(
                   'index': tokenIndex,
                   'docIndex': token.i,
                   'text': token.text,
-                  'punct': token.is_punct,
-                  'metadata': {}
+                  'punct': token.is_punct
```

Comment:
did you decide to map metadata to the other data model? not to a segment?

Reply:
no, it's just that originally I had some of the metadata in a sub object. In the end I made all metadata direct properties of the segment. I went back and forth on how to do this so the code wasn't quite the same everywhere due to my wishy washy refactoring.